workitem/workitem_repository.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/fabric8-services/fabric8-wit/rendering"
 	"github.com/fabric8-services/fabric8-wit/space"
+	numbersequence "github.com/fabric8-services/fabric8-wit/workitem/number_sequence"
```

Comment:
This alias is not necessary as the package name is `numbersequence`.

Reply:
May be that directory name can be changed for better readability.

Reply:
right, thanks !